Source/Other/Extensions/NumberFormatterExtension.swift

```diff
@@ -9,11 +9,16 @@
 import Foundation
 
 extension NumberFormatter {
-
 	@objc public static var decimalStyleFormatter: NumberFormatter = {
-			
 		let formatter = NumberFormatter()
 		formatter.numberStyle = .decimal
 		return formatter
 	}()
+
+	@objc public static func formatter(_ format: String) -> NumberFormatter {
```

Comment:
I know I'm annoying (sorry!) but `formatter(format:` would be better so it's `formatterWithFormat` in Objective-C... for newcomers and more junior developers this wouldn't say anything what the parameter means :)

Reply:
Fixed. And no problem! Happy to receive feedback on the changes :)